openlibrary/plugins/openlibrary/pages/config_edition.page

```diff
@@ -159,8 +159,8 @@
             "label": "Hathi Trust"
         },
         {
-            "url": "http://discovery.lib.harvard.edu/?itemid=|library/m/aleph|@@@",
-            "website": "http://harvard.edu",
+            "url": "https://hollis.harvard.edu/primo_library/libweb/action/display.do?doc=HVD_ALEPH@@@",
```

Comment:
Harvard introduces a new Hollis on 12 March, so maybe hold off on this.  It seems from the preview that its permalinks will not support https, but that may be transitional. Form will be http://id.lib.harvard.edu/aleph/@@@/catalog